docs/docsite/rst/getting_started/get_started_ansible.rst

```diff
@@ -0,0 +1,70 @@
+.. _get_started_ansible:
+
+*******************
+Start using Ansible
+*******************
+
+Start automating with Ansible in a few easy steps.
+
+#. Install Ansible. Visit the :ref:`installation guide<installation_guide>` for complete details.
+
+   .. code-block:: bash
+
+      python3 -m pip install --user ansible
+
+#. Create an inventory by adding the IP address or fully qualified domain name (FQDN) of one or more remote systems to ``/etc/ansible/hosts``.
```

Comment:
Why do we advise users to create inventory in `/etc/ansible/hosts` (scattered over the filesystem), when we discussed we should keep all stuff (inventory, configfile, playbooks, etc) in one project directory in $HOME. Or that work is part of another issue/PR?